shared/src/main/scala/amf/client/model/document/BaseUnit.scala

```diff
@@ -16,7 +16,7 @@ import scala.scalajs.js.annotation.JSExportAll
 
 /** Any parsable unit, backed by a source URI. */
 @JSExportAll
-trait BaseUnit extends AmfObjectWrapper with PlatformSecrets {
+trait BaseUnit extends AmfObjectWrapper with PlatformSecrets with RdfExportable {
```

Comment:
RdfModel trait wasn't exported to JS. That is why I extracted all things related to the RdfModel out of the BaseUnit